README.md

```diff
@@ -136,14 +135,121 @@ make test-ts-binary-rococo
 ./scripts/run-ts-test.sh rococo
 ```
 
-Remember to run the clean-up afterwards.
+Remember to run the clean-up afterward.
 
-## extend the leasing period
+## License
 
-The default leasing duration for parachain is 1 day, in case you want to extend it (even after it's downgraded to parathread), simply do a `forceLease` via sudo, it should be upgraded to parachain soon again and start to produce blocks.
+GPLv3
+
+
+## How to Build and Run Parachain and Tee-worker
+
+## Preparation
+
+- Env: [Setup **SGX TEE** Environment](https://web3builders.notion.site/Setup-SGX-TEE-Environment-68066770831b45b7b632e682cf159477?pvs=4) 
+
+## Build
+
+The `Litentry-worker` needs to be built before the launch of the entire system
+
+```
+git clone https://github.com/litentry/litentry-parachain
+cd ./litentry-parachain/tee-worker
+source /opt/intel/sgxsdk/environment
+SGX_MODE=SW make
+```
+
+## Launch
+
+### 1. Start a local docker setup
+
+In order to create a local docker setup, you can run the following command
+```
+./local-setup/launch.py --config ./local-setup/github-action-config-one-worker.json
+```
+This will create three docker containers, 2 Relay Chain Validators, and 1 Parachain Collator. However, it will use the default ports as present in .env.dev. If you want to run the system by offsetting the default ports, you can run this command instead:
+
+```
+./local-setup/launch.py --config local-setup/development-worker.json --offset 100
+```
+This will run the same containers and use the offset value of 100.
+
+### 2. Start a local binary setup
+
+In order to create a local binary setup, using default ports, you can run the following command:
+```
+/local-setup/launch.py --config ./local-setup/github-action-config-one-worker.json --parachain local-binary
+```
+
+If you want to launch the same system by offsetting the port values, you can use this command: 
+```
+./local-setup/worker-log-level-config.toml 100 
```

Comment:
This is incorrect, Check the notion page again, I've updated this command. it should be the following: 
```
/local-setup/launch.py --config ./local-setup/github-action-config-one-worker.json --parachain local-binary --offset 100
```

Reply:
Updated.